tests/aws/services/cloudformation/v2/ported_from_v1/resources/test_events.py

```diff
@@ -18,7 +18,9 @@
 )
 
 
-@pytest.mark.skip(reason="CFNV2:ReferenceDotSyntax")
+@pytest.mark.skip(
+    reason="CFNV2:Other resource name conflict with another test case resource in this suite"
+)
```

Comment:
I guess because the deletion is not implemented yet, so we are leaving resources lying around.